app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        difference = self.clean_power - car.clean_mark
+        upper_part = car.comfort_class * difference * self.average_rating
+        return round(upper_part / self.distance_from_city_center, 1)
+
+    def serve_cars(self, car_list: Car) -> float:
```

Comment:
It's incorrect type annotation

```suggestion
    def serve_cars(self, car_list: list[Car]) -> float:
```